src/lsp/MessageHandlers.re

```diff
@@ -345,10 +345,9 @@ let handlers: list((string, (state, Json.t) => result((state, Json.t), string)))
           CodeLens.forOpens(extra)
         } : lenses;
 
-        let showDependencies = state.settings.dependenciesCodelens;
-        let lenses = showDependencies ? [("Dependencies: " ++ String.concat(", ",
-          List.map(SharedTypes.hashList(extra.externalReferences), fst)
-        ), topLoc), ...lenses] : lenses;
+        let depsList = List.map(SharedTypes.hashList(extra.externalReferences), fst)
+        let depsString = depsList == [] ? "-" : String.concat(", ", depsList)
+        let lenses = (state.settings.dependenciesCodelens==true) ? [("Dependencies: " ++ depsString, topLoc), ...lenses] : lenses;
```

Comment:
```suggestion
        let depsString = depsList == [] ? "This file has no dependencies" : ("Dependencies: " ++ String.concat(", ", depsList))
        let lenses = (state.settings.dependenciesCodelens==true) ? [(depsString, topLoc), ...lenses] : lenses;
```

Reply:
could we keep the additional text output minimalistic in this case please, since it affects the file header line and people read this so often; less is more, less prevents distracting.

Reply:
hmm sure that's fine -- I just think the " - " is almost as confusing as not having anything. What about `Dependencies: [none]`?

Reply:
This would be nice :)